### DISCUSSION TITLE
How to use transactions over multiple databases

### DISCUSSION THREAD

**Discussion by tayyabferozi** [Automattic/mongoose#9882] [Q&A] [answered]
Basically I am creating a reference of one user on two differenct dabases hosted at different places. I want to save the reference in one database if and only if the user was saved on the other database.

Is there a way to shared one mongoose session over two different dabases? I am using axios library to connect between two node.js hosted servers

**Comment by vkarpov15** [comment, marked as answer]
Unfortunately not, MongoDB doesn't support transactions across separate database clusters. You'd have to implement a transaction yourself using something like this: https://www.docs4dev.com/docs/en/mongodb/v3.6/reference/tutorial-perform-two-phase-commits.html
